src/main/java/uk/gov/hmcts/darts/arm/service/impl/ArmBatchProcessResponseFilesImpl.java

```diff
@@ -166,7 +203,7 @@ private void resetArmStatusForUnprocessedEods(String manifestName) {
         }
     }
 
-    private void processResponseFileByHashcode(BatchInputUploadFileFilenameProcessor batchUploadFileFilenameProcessor) {
+    private void processResponseFileByHashcode(BatchInputUploadFileFilenameProcessor batchUploadFileFilenameProcessor, String manifestName) {
```

Comment:
is this new param needed?

Reply:
Its used for logging. There is an issue with tracing log messages via dynatrace as it uses different things in the logs.

Reply:
The `manifestName` parameter isn't used anywhere in the `processResponseFileByHashcode()` method though?

Reply:
It is on line 221 in the log message